### DISCUSSION TITLE
i has update 1.0.5, but can't find the plot which had ploted before

### DISCUSSION THREAD

**Discussion by githubchia99** [Chia-Network/chia-blockchain#2034] [Q&A]
i had done and  update the charse to 1.0.5, but the new charse can't find the plots which had ploted before, how to set the new version to find the ploted farm

**Comment by dorkmo** [comment]
are you in the GUI? you can click the 3 dots next to "add plot" on the farm page, then click add directory

**Reply by githubchia99** [reply]
i do  ,but it's not work,it's still can't find the plots

**Reply by jjhesk** [reply]
are you farming the previous plotted files?

**Reply by githubchia99** [reply]
no  the previous plotted files has  done.but can't find it ,and not display it when update ther version

**Reply by wenbsu** [reply]
I have the same problem

**Comment by cccat6** [comment]
Are you using the plots for testnet only? Mainnet supports k>=32 plots.

**Reply by githubchia99** [reply]
two plots  K>=33 .the other is k>=32  。

**Reply by githubchia99** [reply]
the other  10 plots k>=32

**Reply by cccat6** [reply]
Are you sure the dir you added in actually contains the plots file?
If the dir contains the plots file, it should appear in the plots list. Even the plot files are damaged, they will show in the list.
![image](https://user-images.githubusercontent.com/22387156/115330696-29fde100-a1c7-11eb-9580-e1630b96343e.png)
![image](https://user-images.githubusercontent.com/22387156/115330715-32561c00-a1c7-11eb-86c7-0b0e56cd88d2.png)
![image](https://user-images.githubusercontent.com/22387156/115330732-397d2a00-a1c7-11eb-948a-3b3daf5265dd.png)
![image](https://user-images.githubusercontent.com/22387156/115330915-88c35a80-a1c7-11eb-870a-7ea3a2c8a0cc.png)

**Reply by githubchia99** [reply]
yes i'm sure . i do it like the pic you show

**Reply by cccat6** [reply]
I guess you need to wait for the maintainer's reply. I can't help in this case since it is strange.

**Reply by githubchia99** [reply]
when i add a plot .the local harvest plots display the words " Failed to open (invalid plots)
These plots are invalid, you might want to delete them."

**Reply by githubchia99** [reply]
and thank you

**Reply by wenbsu** [reply]
I have the same problem
![image](https://raw.githubusercontent.com/wenbsu/mypicture/main/addDirectory.bmp)
![image](https://raw.githubusercontent.com/wenbsu/mypicture/main/previewDirectory.bmp)
![image](https://raw.githubusercontent.com/wenbsu/mypicture/main/invalidation.bmp)
